docs/glossary.md

```diff
@@ -0,0 +1,17 @@
+# Glossary
+
+Quick reference for tools.
+
+| Symbol | Tool | Abbreviation | Documentation |
+|--------|------|--------------|---------------|
+|        |   [OpenEnergyPlatform](https://openenergy-platform.org/)   |       OEP      |               |
+|        |    OEMetadata  |       OEM        |        [ docs ](https://github.com/OpenEnergyPlatform/oemetadata/blob/develop/metadata/latest/metadata_key_description.md#oemetadata---key-description)       |
+|        |  OEdatamodel    |        OED      |       [ docs ](https://github.com/sedos-project/oedatamodel)      |
+| | [OEdatamodel-API](https://modex.rl-institut.de/upload_datapackage/) | | [ docs ](https://modex.rl-institut.de/) |
+|    🌾   |   OpenAnnotationTool   |       OAT       |               |
+|        |   [oemetadata builder](https://meta.rl-institut.de/meta_creator/151)    |              |               |
+| 🌠 | [databus](https://energy.databus.dbpedia.org/)  | | |
+| 📙 | [Protégé](https://protege.stanford.edu/) | | [docs](http://protegeproject.github.io/protege/) |
+| | | | |
+
+
```

Comment:
Altogether, I believe that this documentation offers an extensive support for all data related activities. Next to background and understanding sections, a lot of links to further information from the OEP is provided on one hand, but on the other hand compact recipes are given for specific tasks. Especially the annotation section is very nice with its examples. Maybe some more detailed examples in other sections would further help. 
I think the documentation will be very helpful for future SEDOS tasks!

Reply:
Thanks for the thorough review and helpful remarks!